plugins/gitea/personal_access_token.go

```diff
@@ -0,0 +1,120 @@
+package gitea
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+	"gopkg.in/yaml.v2"
+)
+
+var configPath string = "~/.config/tea/config.yml"
+
+func PersonalAccessToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.PersonalAccessToken,
+		DocsURL:       sdk.URL("https://gitea.com/user/settings/applications"),
+		ManagementURL: nil,
```

Comment:
For the DocsURL we expect a documentation page that is not behind a login. The management URL is the in-product page where you can create and manage the credential(s).

See for example the [GitLab plugin](https://github.com/1Password/shell-plugins/blob/main/plugins/gitlab/personal_access_token.go) for an example.

```suggestion
		DocsURL:       sdk.URL("https://docs.gitea.io/en-us/command-line/"),
		ManagementURL: sdk.URL("https://gitea.com/user/settings/applications"),
```

Reply:
I had a bit of indecision about this. The Gitea command-line section in their docs is for running/administering the server. The documentation for users in general is fairly sparse, the only place I've been able to find documentation about tea is that Readme in its repo. However there is no mention of Tokens. Perhaps https://docs.gitea.io/en-us/api-usage/#generating-and-listing-api-tokens would be better, there is a small mention at the bottom of that section about generating a Token through the UI.

Reply:
I like your suggestion to link to the docs section on generating and listing API tokens 👍 That's indeed more useful for end-users than the command-line doc I proposed.